src/methods/logout.js

```diff
@@ -1,14 +1,9 @@
 import emitter from './../lib/emitter'
 
-export default self => {
-  // just clears current session and sessionStorage
-  self.session = {}
-
-  window.sessionStorage.removeItem('my_id')
-  window.sessionStorage.removeItem('access_token')
-  window.sessionStorage.removeItem('expires')
-  window.sessionStorage.removeItem('username')
+export default (self, session) => {
+  // just clears current session
+  session.access_token = ''
   // emit logout event
   emitter.emit('logout', self)
-  return self
+  return session
```

Comment:
acho melhor retornar o self mesmo, mas possibilitar chain...

```suggestion
  return self
```